src/components/visualizations/chart/highcharts/plugins/getBubbleRadius.ts

```diff
@@ -0,0 +1,20 @@
+// (C) 2007-2018 GoodData Corporation
+
+import isNil = require("lodash/isNil");
+
+export function getBubbleRadius(Highcharts: any) {
+    const wrap = Highcharts.wrap;
+
+    if (Highcharts.seriesTypes.bubble) {
+        wrap(Highcharts.seriesTypes.bubble.prototype, "getRadius", function(proceed: any, ...args: number[]) {
+            const [minSize, maxSize, value] = args.slice(2, 5);
+            let radius = proceed.apply(this, args);
+            let pos;
+            if (isNaN(value) && isNil(radius)) {
+                pos = Math.sqrt(0.5);
+                radius = Math.ceil(minSize + pos * (maxSize - minSize)) / 2;
+            }
+            return radius;
+        });
+    }
```

Comment:
please consider this, I want it much explicit as possible
and it's nice to have more comments about `Math.sqrt(0.5)`
thanks
```
if (Highcharts.seriesTypes.bubble) {
        wrap(Highcharts.seriesTypes.bubble.prototype, "getRadius", function(proceed: any, _zMin:number, _zMax: number, minSize: number, maxSize: number, value: number) {
            const radius = proceed.apply(this, [_zMin, _zMax, minSize, maxSize, value]);
            if (isNaN(value) && isNil(radius)) {
                const pos = Math.sqrt(0.5);
                return Math.ceil(minSize + pos * (maxSize - minSize)) / 2;
            }
            return radius;
        });
    }
```

Reply:
comments is added

Reply:
done

Reply:
so we should have const for 0.5 and add comment right there.
and please consider not using `let pos` anymore. We can use this constant directly.